client/src/app/account-card/account-card.component.scss

```diff
@@ -0,0 +1,51 @@
+@use "../shared/variables.scss" as var;
+
+.account-container {
+  margin-left: 5rem;
+  margin-top: 2rem;
+  position: relative;
+  width: max-content;
+}
+
+.account-card {
+  width: 18rem;
+  height: 8rem;
+  background-image: linear-gradient(
+    to right bottom,
+    #f4497d,
+    #e43d95,
+    #c93dac,
```

Comment:
here should be 5 constants!

Reply:
i'll use it only once on the cards therefore i left it but i'll change it